documentation/collectionrules/TriggerShortcuts.md

```diff
@@ -0,0 +1,201 @@
+# Trigger Shortcuts
```

Comment:
Nit: **t**rigger**s**hortcuts.md

Reply:
You probably need to delete this file and then readd it again as lowercase. Otherwise, it'll keep the original casing.